src/linearmixedmodel.jl

```diff
@@ -673,6 +673,22 @@ function ranef(m::LinearMixedModel{T}; uscale = false, named = false) where {T}
     vnmd
 end
 
+function retbl(mat, trm)
+    merge(
+        NamedTuple{(trm.trm.sym,)}((trm.levels,)),
```

Comment:
I suspect this will run into issues with grouping variables that are interactions (see #361).

Reply:
Thanks for catching that.